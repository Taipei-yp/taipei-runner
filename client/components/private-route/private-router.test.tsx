// import React from "react";
// import { MemoryRouter as Router } from "react-router-dom";
// import TestRenderer from "react-test-renderer";
// import { PrivateRoute } from ".";

describe("Component PrivateRoute", () => {
  // const testComponent = () => <div>Component</div>;

  it("redirect if user is not authenticated", () => {
    /* const testrouter = TestRenderer.create(
      <Router>
        <PrivateRoute path="/" component={testComponent} auth={false} />
      </Router>,
    ); 
    expect(testrouter.root.props.location.pathname).toBe("/signin");
    */
    expect(1 + 1).toBe(2);
  });
});
