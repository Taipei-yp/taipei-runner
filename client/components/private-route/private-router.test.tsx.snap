// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component PrivateRoute render if user is authenticated 1`] = `
<div>
  Component
</div>
`;
